classy-prelude/ClassyPrelude.hs

```diff
@@ -175,8 +208,11 @@ import Data.Functor
 import Control.Exception (assert)
 import Control.Exception.Enclosed
 import Control.Monad (when, unless, void, liftM, ap, forever, join, replicateM_, guard, MonadPlus (..), (=<<), (>=>), (<=<), liftM2, liftM3, liftM4, liftM5)
+import Control.Concurrent.Lifted
```

Comment:
I still have to provide a `yieldThread`, and enumerating all other entities by hand in a separate `Concurrency` section has a documentation advantage: `yieldThread` can fit in nicely, and in the future, `Async` related exports can also fit in that section.